app/jobs/enqueue_publishers_for_payout_job.rb

```diff
@@ -2,11 +2,13 @@
 class EnqueuePublishersForPayoutJob < ApplicationJob
   queue_as :scheduler
 
-  def perform(should_send_notifications: false, final: true, payout_report_id: "", publisher_ids: [])
+  def perform(should_send_notifications: false, final: true, manual: false, payout_report_id: "", publisher_ids: [])
     Rails.logger.info("Enqueuing publishers for payment.")
 
     if publisher_ids.present?
       publishers = Publisher.where(id: publisher_ids)
+    elsif manual
+      publishers = Publisher.partner
     else
       publishers = Publisher.with_verified_channel
```

Comment:
My impression is that the partner should be excluded from this segment. @nvonpentz would you confirm?

Reply:
This could be true in the future for specified `orgs` but in the current use case, partner's will be paid out as regular publishers with regards to their channels.